roles/openshift_logging_elasticsearch/tasks/rolling_cluster_restart.yml

```diff
@@ -73,6 +73,9 @@
     -XPUT 'https://logging-{{ _cluster_component }}.{{ openshift_logging_elasticsearch_namespace }}.svc:9200/_cluster/settings'
     -d '{ "transient": { "cluster.routing.allocation.enable" : "all" } }'
   register: _cluster_enable_output
+  until: _cluster_status.rc == 0
```

Comment:
this should be `_cluster_enable_output.rc`

Reply:
Of course. My mistake. Fixed!